inc/class-wpseo-utils.php

```diff
@@ -669,16 +669,13 @@ public static function trim_nbsp_from_string( $string ) {
 	 * @return bool
 	 */
 	public static function is_valid_datetime( $datetime ) {
+		static $date_helper;
```

Comment:
This method is now a wrapper / utility which didn't exist before but still has the notation `@since 2.0.0`.
The rpe-existing method with the same name is now moved to `inc/date-helper.php` and has a notation `@since 2.0.0` as well. Wondering if the new wrapper should use a more recent `@since` because, actually, it's "new".